blackbox/docs/appendices/release-notes/3.3.0.rst

```diff
@@ -0,0 +1,160 @@
+has now been deprecated, in favour of the new total count and sum of durations
+metrics.
+
+The ``cluster.graceful_stop.reallocate`` setting has been marked as deprecated.
+This setting was already being ignored, and setting the value to ``false``
+had no effect.
+
+The node decommission using the ``USR2`` :ref:`signals <cli_signals>` has now
+been deprecated in favour of the :ref:`ALTER CLUSTER DECOMISSION
+<alter_cluster_decommission>` statement.
+
+The ``CREATE INGEST`` and ``DROP INGEST`` rules have been marked
+as deprecated. Given that the only implementation (MQTT) was deprecated
+and will be removed, the framework itself will also be removed.
+
+
+Changelog
+=========
+
+Changes
+-------
+
+New Features
+~~~~~~~~~~~~
+
+- Exposed the sum of durations, total, and failed count metrics under the
+  :ref:`query_stats_mbean` for ``QUERY``, ``INSERT``, ``UPDATE``, ``DELETE``,
+  ``MANAGEMENT``, ``DDL``, and ``COPY`` statement types.
+
+- Exposed the sum of statement durations, total, and failed count classified by
+  statement type under the ``sum_of_durations``, ``total_count`` and
+  ``failed_count`` columns, respectively, in the :ref:`sys-jobs-metrics` table.
+
+SQL Improvements
+~~~~~~~~~~~~~~~~
+
+- Added ``current_schemas(boolean)`` scalar function which will return the
+  names of schemas in the ``search_path``.
+
+- Added support for the ``first_value``, ``last_value``, and ``nth_value``
+  window functions as enterprise features.
+
+- Added the ``DROP ANALYZER`` statement to support removal of custom
+  analyzer definitions from the cluster.
+
+- Output the custom analyzer, tokenizer, token_filter, and char_filter
+  definition inside the ``information_schema.routines.routine_definition``
+  column.
+
+- Added support for the ``row_number()`` window function.
+
+- Added support for using any expression in the operand of a ``CASE`` clause.
+
+- Fix quoting of identifiers that contain leading digits or spaces when
+  printing relation or column names.
+
+PostgreSQL Compatibility
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added ``pg_type`` columns: ``typlen``, ``typarray``, ``typnotnull``,
+  and ``typnamespace`` for improved PostgreSQL compatibility.
+
+- Added a ``pg_description`` table to the ``pg_catalog`` schema for improved
+  PostgreSQL compatibility.
+
+- Fixed function resolution for PostgreSQL functions ``pg_backend_pid``,
+  ``pg_get_expr``, and ``current_database`` when the schema prefix
+  ``pg_catalog`` is included.
+
+Database Administration
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added a node check for the JVM version number.
+
+- Added ``ALTER CLUSTER DECOMMISSION <nodeId | nodeName>`` statement that
+  triggers the existing node decommission functionality.
+
+- Changed the trial license introduced in 3.2 to no longer have an expiration
+  date, but instead be limited to three nodes. See :ref:`enterprise_features`.
+
+- The :ref:`usage_data_collector` now includes information about the available
+  number of processors.
+
+Deprecations
```

Comment:
isn't this a duplicate of the above deprecations?

Reply:
Yes, but this is intentional: we have a similar pattern in the 3.2.0 notes https://crate.io/docs/crate/reference/en/latest/appendices/release-notes/3.2.0.html

Essentially, deprecations are very important so they're put in the Upgrade Notes at the top - but theyre still part of the general changelog.